src/components/barnetillegg/LeggTilBarnModal.tsx

```diff
@@ -0,0 +1,213 @@
+                                >
+                                    Etternavn
+                                </Fritekstspørsmål>
+                                <Datospørsmål
+                                    name='svar.barnetillegg.kladd.fødselsdato'
+                                    datoMåVæreIFortid={true}
+                                    validate={datofeltValidator}
+                                >
+                                    Fødselsdato
+                                </Datospørsmål>
+                                <JaNeiSpørsmål
+                                    reverse
+                                    name={`svar.barnetillegg.kladd.oppholdUtenforEØS`}
+                                    validate={barnUtenforEØSValidator}
+                                    hjelpetekst={{
+                                        tittel: 'Hvorfor spør vi om dette?',
+                                        tekst: (
+                                            <>
+                                                <span>
+                                                    Hvis barnet ditt oppholder seg utenfor EØS i tiltaksperioden kan det ha betydning for
+                                                    din rett til barnetillegg.
+                                                </span>
+                                                <span style={{display: 'block', marginTop: '1rem'}}>
+                                                <Link
+                                                    href="https://www.nav.no/no/person/flere-tema/arbeid-og-opphold-i-utlandet/relatert-informasjon/eos-landene"
+                                                    target="_blank"
+                                                >
+                                                    Du kan lese mer om hvile land som er med i EØS her.
+                                                    <ExternalLinkIcon title="a11y-title"/>
+                                                </Link>
+                                                </span>
+                                            </>
+                                        ),
+                                    }}
+                                >
+                                    Oppholder barnet ditt seg utenfor EØS i tiltaksperioden?
+                                </JaNeiSpørsmål>
+                            </>
+                    <Button
+                        onClick={lukkModal}
+                        className={styles.knappAvbrytModal}
+                        variant="secondary"
+                    >
+                        Avbryt
+                    </Button>
+                    <div/>
+                    <Button
+                        onClick={async (e) => {
+                            e.preventDefault()
+                            const harIngenValideringsfeil = await trigger('svar.barnetillegg.kladd')
```

Comment:
Vil ikke type=submit på button (og bruk av onSubmit på et `<form>`-element) sørge for at valideringen trigges automatisk, eller er det noe jeg ikke ser her? 🤔